Tests/SwiftParserTest/ExpressionTests.swift

```diff
@@ -380,7 +397,13 @@ final class ExpressionTests: XCTestCase {
           trailingComma: .commaToken()
         )
       ),
-      substructureAfterMarker: "1️⃣"
+      substructureAfterMarker: "1️⃣",
+      fixedSource: """
```

Comment:
We shouldn't need `fixedSource` here, right? Actually we might want to make it an assertion failure if you provide a `fixedSource` without specifying any expected diagnostics.

Reply:
Good catch! 

Added an assertion